src/web/mod.rs

```diff
@@ -104,6 +111,7 @@ pub fn run(soup: Blender) -> HttpResult<Listening> {
     Server {
             handlers: router,
             host: 8080.into(),
+            global: Global::from(Box::new(Mutex::new(soup))),
```

Comment:
Hmm.. The docs say that [it requires](https://doc.rust-lang.org/nightly/core/convert/trait.From.html#tymethod.from) a `Box`, so seems like you'd need `Box::new(soup)`. But the `Mutex` shouldn't be necessary?